File: Skeilink/check_braces.py
with open('src/App.jsx', 'r') as f:
    lines = f.readlines()

depth = 0
last_func = ""
for i, line in enumerate(lines):
    if "function " in line:
        last_func = line.strip()
    for c in line:
        if c == '{': depth += 1
        if c == '}': depth -= 1
    if depth < 0:
        print(f"Negative depth at line {i+1}: {line.strip()}")

print("Final depth:", depth)

File: Skeilink/check_braces.js
const fs = require('fs');
const lines = fs.readFileSync('src/App.jsx', 'utf-8').split('\n');
let depth = 0;
let lastFunc = "";
for(let i=0; i<lines.length; i++) {
  const line = lines[i];
  if (line.includes('function ')) lastFunc = line.trim();
  for (let c of line) {
    if (c === '{') depth++;
    if (c === '}') depth--;
  }
}
console.log("Final depth:", depth);
